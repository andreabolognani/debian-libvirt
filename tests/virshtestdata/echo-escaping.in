echo         a A 0 + * . \' / ? =  < > &
echo --xml   a A 0 + * . \' / ? =  < > &
echo --shell a A 0 + * . \' / ? =  < > &
echo ""
echo --shell
echo --xml
echo --shell
echo ''
echo --shell ""
echo --xml ''
echo --shell ""''
