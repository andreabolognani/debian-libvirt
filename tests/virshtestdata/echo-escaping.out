a A 0 + * . ' / ? = < > &
a A 0 + * . &apos; / ? = &lt; &gt; &amp;
a A 0 + '*' . ''\''' / '?' = '<' '>' '&'





''

''
